spec/rails_helper.rb

```diff
@@ -7,6 +7,7 @@
 require 'rspec/rails'
 # Add additional requires below this line. Rails is not loaded until this point!
 require 'devise'
+require_relative 'support/controller_macros'
```

Comment:
[as per the Devise documentation](https://github.com/heartcombo/devise/wiki/How-To:-Test-controllers-with-Rails-(and-RSpec)#controller-specs).